src/App.tsx

```diff
@@ -1,61 +1,125 @@
+import React, { useState } from 'react';
+
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+
+import { TodoList } from './components/TodoList';
+
+function getUser(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+export const todos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
 
 export const App = () => {
+  const [userId, setUserId] = useState(0);
+  const [title, setTitle] = useState('');
+  const [currentTodos, setCurrentTodos] = useState(todos);
+  const [isUserSelected, setIsUserSelected] = useState(true);
+  const [isTitleEntered, setIsTitleEntered] = useState(true);
```

Comment:
Do you really need to keep track of title and if the title is entered? You can get its boolean value instead.